types/config.d.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable strict */
```

Comment:
is anyone else having this problem in vscode where eslint is marking 100% of our .d.ts files as failing because of the strictness? started happening for me once we switched to the typescript parser in eslint

Reply:
yup same. vscode's really unhappy with my .d.ts files.